i18n/qqq.json

```diff
@@ -8,5 +8,10 @@
 	"github-desc": "{{desc|name=GitHub|url=https://www.mediawiki.org/wiki/Extension:GitHub}}",
 	"github-par-filename": "Label for the contents of the \"file name\" parameter\n{{Identical|Filename}}",
 	"github-par-branchname": "Label for the contents of the \"branch name\" parameter",
-	"github-par-reponame": "Label for the contents of the \"repository name\" parameter"
+	"github-par-reponame": "Label for the contents of the \"repository name\" parameter",
+	"github-par-lang": "Label for the contents of the \"lang\" parameter",
+	"github-par-line": "Label for the contents of the \"line\" parameter",
+	"github-par-start": "Label for the contents of the \"start\" parameter",
+	"github-par-highlight": "Label for the contents of the \"highlight\" parameter",
+	"github-par-inline": "Label for the contents of the \"inline\" parameter",
```

Comment:
It's be nice to have the descriptions be useful to anyone that does not know the SyntaxHighlight extension